### PR TITLE
Web UI: design updates and tweaks

### PR DESCRIPTION
**Pull request by simonwo**
See individual commits for more info.

image
image

### CODE REVIEW

**Comment by frrist** [comment]
> [Show jobs table in date order and with relative dates.](https://github.com/bacalhau-project/bacalhau/pull/3052/commits/3c5098f9269b36b76bea27250a20d07883e02fbe)
 
❤️  Does this imply the order of jobs will remain consistent across refreshes? This might be something we want to bake into the API method we are calling as well.

File: webui/src/styles/_variables.scss [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // _variables.scss
 
 $bgd-color: #fff8f8;
-$primary-blue: #001c70;
+$primary-blue: #05f;
```
Comment (XandraMcC) [inline webui/src/styles/_variables.scss:4]: What is the reason for the colour change? Would be good to consult @Laurahohmann on this

**Comment by XandraMcC** [comment]
Added new icons with new colour scheme + fixed CSS issue with Jobs Table.

File: webui/src/layout/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -13,35 +14,39 @@ interface SidebarProps {
...
-            <span className={styles.menuText}>Nodes Dashboard</span>
-          </Link>
-        </div>
-        <div className={styles.menuItem}>
-          {/* Settings Icon */}
-          <Link to="/Settings" className={styles.link}>
-            <img src={settingsIcon} alt="Icon" width={20} height={20} />
-            <span className={styles.menuText}>Settings</span>
+        {links.map(link =>
+          <Link to={link.path} className={styles.menuItem} data-selected={document.location.pathname.startsWith(link.path)} title={link.title}>
```
Comment (XandraMcC) [inline webui/src/layout/Sidebar/Sidebar.tsx:45]: Need to add `key={link.title}` to get round compilation error:

```
<Link 
          key={link.title}
          to={link.path} 
          className={styles.menuItem} 
          data-selected={document.location.pathname.startsWith(link.path)} 
          title={link.title}
        >
```

**Review by XandraMcC — APPROVED** [review]

**Pull request merged**